src/main.js

```diff
@@ -95,6 +96,26 @@ function getNLSConfiguration() {
 	return { locale: initialLocale, availableLanguages: {} };
 }
 
+function migrateUserDataDir(newDir) {
+	var oldDir = path.join(process.env.HOME, '.config', product.nameShort);
+	if (!fs.existsSync(oldDir)) {
+		return;
+	}
+	mkdirp(path.dirname(newDir));
+	fs.renameSync(oldDir, newDir);
```

Comment:
We have to verify on all platforms that renaming the chrome data folder like this keeps all local storage functional!